src/app/modules/consultations/consultation-profile/consultation-public-responses/consultation-public-responses.component.ts

```diff
@@ -13,11 +14,13 @@ export class ConsultationPublicResponsesComponent implements OnInit, AfterViewCh
 
   profileData: any;
   responseList: any;
-  responseQuestions: any;
```

Comment:
@Nishanthh This shouldn't be deleted, right? We will still need to display the response questions in the UI